src/gltf-component-mappings.js

```diff
@@ -72,3 +72,22 @@ AFRAME.GLTFModelPlus.registerComponent("nav-mesh", "nav-mesh", (el, _componentNa
   // nav-mesh-helper will query for it later.
   el.setAttribute("nav-mesh");
 });
+
+AFRAME.GLTFModelPlus.registerComponent("pinnable", "pinnable");
+
+AFRAME.GLTFModelPlus.registerComponent("media", "media", (el, componentName, componentData) => {
+  if (componentData.id) {
+    el.setAttribute("networked", {
+      template: "#interactable-media",
+      owner: "scene",
+      lastOwnerTime: 1,
+      networkId: componentData.id
+    });
+  }
+
+  el.setAttribute("media-loader", { src: componentData.src, resize: true, resolve: true });
+
+  if (componentData.pageIndex) {
```

Comment:
should be fine to just always include ` index: componentData.pageIndex` in the previous setAttribute

Reply:
the component below is `media-pager` -- so I think you may have just misread this. `media-loader` won't deal with `index` right?